script.js

```diff
@@ -75,26 +75,36 @@ function updateCanvas() {
     drawCharge(charges[i]);
   }
 }
+
 // creates paths array of the form [[[x11, y11], [x12, y12], ...], [[x21, y21], [x22, y22], ...], ...]
 function calculateFieldLines() {
   let paths = [];
   for (let i = 0; i < charges.length; i++) {
-    let startAngle = TAU * Math.random();
-    let angleStep = TAU / Math.floor(1 + Math.abs(charges[i][2]));
-    for (let j = 0; j <= Math.abs(charges[i][2]); j++)
-      paths.push(calculatePath([charges[i][0] + Math.sqrt(Math.abs(charges[i][2])) * Math.cos(startAngle + j * angleStep), charges[i][1] + Math.sqrt(Math.abs(charges[i][2])) * Math.sin(startAngle + j * angleStep), charges[i][2]]));
+    let angle = TAU * Math.random();
```

Comment:
Oh, I should have recognized tau when I commented on it the other day. Btw, you should probably define it (up at line 4 or whatever as:

```
const TAU = Math.PI * 2;
```